rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -110,7 +110,7 @@ D3D12_STENCIL_OP get_stencil_op(u32 op) noexcept
 	case CELL_GCM_INCR: return D3D12_STENCIL_OP_INCR;
 	case CELL_GCM_DECR: return D3D12_STENCIL_OP_DECR;
 	case CELL_GCM_INVERT: return D3D12_STENCIL_OP_INVERT;
-	case CELL_GCM_INCR_WRAP:
+	case CELL_GCM_INCR_WRAP: return D3D12_STENCIL_OP_INCR;
```

Comment:
The hack fix should probably at least still log that a hack is being used too, otherwise its gonna be a nightmare to track down why something is screwed. @Zangetsu38 you should really make a test app for PS3 that tests what this really does, then you'll see its wrong.